colossus/src/main/scala/colossus/protocols/http/Header.scala

```diff
@@ -138,7 +190,7 @@ class HttpHeaders(private val headers: Array[HttpHeader]) {
     case other => false
   }
 
-  override def toString = "[" + headers.map{_.toString}.mkString(" ") + "]"
+  override def toString = "[" + toSeq.map{_.toString}.mkString(" ") + "]"
```

Comment:
`toString` should never be called in a hot path, it's mainly just for debugging.  colossus itself certainly doesn't use it.